sys/net/gnrc/link_layer/gomach/gomach_internal.c

```diff
@@ -939,17 +940,19 @@ void gnrc_gomach_process_preamble_ack(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt)
                         gomach_preamble_ack_hdr->phase_in_us;
 
     if (phase_us < 0) {
-        phase_us += GNRC_GOMACH_SUPERFRAME_DURATION_US;
+        phase_us += CONFIG_GNRC_GOMACH_SUPERFRAME_DURATION_US;
     }
 
-    if (((uint32_t)phase_us > (GNRC_GOMACH_SUPERFRAME_DURATION_US - GNRC_GOMACH_CP_MIN_GAP_US)) ||
-        ((uint32_t)phase_us < GNRC_GOMACH_CP_MIN_GAP_US)) {
+    if (((uint32_t)phase_us > (CONFIG_GNRC_GOMACH_SUPERFRAME_DURATION_US - \
```

Comment:
Here and bellow `\` are not needed

Reply:
Also the indentation looks a bit confusing. Could `CONFIG_GNRC_GOMACH_CP_MIN_GAP_US` be indented to the level of `CONFIG_GNRC_GOMACH_SUPERFRAME_DURATION_US`?

Reply:
Done.